indico/migrations/versions/20200623_1718_05f227f4b938_add_internal_data_to_event_logs.py

```diff
@@ -0,0 +1,28 @@
+"""Add internal data to event logs
+
+Revision ID: 05f227f4b938
+Revises: c0fc1e46888b
+Create Date: 2020-06-23 17:18:05.171784
+"""
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+
+# revision identifiers, used by Alembic.
+revision = '05f227f4b938'
+down_revision = 'b6dd0a4ed40d'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('logs', sa.Column('internal_data', postgresql.JSONB(), nullable=False, server_default='{}'),
```

Comment:
Just `metadata` is reserved by SQLAlchemy, but `meta` would work. I actually like this, especially because we could then use `meta.FIELD=...` in the URL!